IPython/terminal/ipapp.py

```diff
@@ -154,6 +154,7 @@ def make_report(self,traceback):
 
 aliases = dict(base_aliases)
 aliases.update(shell_aliases)
+aliases['term-title-format'] = 'TerminalInteractiveShell.term_title_format'
```

Comment:
Actually, I don't see that flag when I remove the alias, both in the CLI help and the profile config.  Is there something else I'm missing to get it to auto-generate the flag?